ykbh/src/lib.rs

```diff
@@ -0,0 +1,310 @@
+#![feature(test)]
+extern crate test;
+
+use ykpack::{self, Local, Terminator, Statement, IPlace, Constant, ConstantInt, UnsignedInt};
+use yktrace::sir::SIR;
+use std::convert::TryInto;
+
+const ZST: Object = Object { data: Vec::new() };
+
+#[derive(Clone, Debug)]
+/// A data object, storing bytes.
+pub struct Object {
```

Comment:
So is there one `Object` for each `Local`?

Reply:
Yeah, there used to be more inside the Object. As noted in the PR description, this is not very optimal is gonna change very soon.

Reply:
I see. Perhaps later we can use a name which captures the relationship to a local. Maybe `LocalStorage` or something?

Reply:
If it's just a name change, I can update it in this PR.

Reply:
Maybe `LocalBytes`?

Reply:
Fine with me.